### ISSUE TITLE
Corrupted Statusbar tips cache causes panic

### ISSUE THREAD

**Issue by raphCode** [zellij-org/zellij#1944]
If the file `/tmp/zellij-<uid>/status-bar-tips.cache` is corrupted, zellij won't start up or existing sessions crash when a new tab is opened:
```
Error occurred in server:

  × Thread 'wasm' panicked.
  ├─▶ Originating Thread(s)
  │       1. ipc_server: NewClient
  │       2. pty_thread: NewTab
  │       3. screen_thread: NewTab
  │       4. plugin_thread: Load
  │
  ├─▶ At zellij-server/src/wasm_vm.rs:670:5
  ╰─▶
A panic occured in a plugin
PanicInfo {
    payload: Any { .. },
    message: Some(
        called Result::unwrap() on an Err value: Serde(Error("trailing characters", line: 1, column: 238)),
    ),
    location: Location {
        file: "default-plugins/status-bar/src/tip/utils.rs",
        line: 40,
        col: 83,
    },
    can_unwind: true,
}
```
I experienced the corruption sometimes when start multiple zellij sessions in parallel.

**Comment by imsnif** [comment]
I think @har7an already solved this, right?

**Comment by har7an** [comment]
@imsnif I don't think so. A quick look at the code there suggests I didn't improve the error handling over simple unwrap statements (yet). My other "stability" PR was concerned with the `~/.cache/zellij` folder, but the status bar tips cache is located in `/tmp`, so that won't help here I'm afraid. @raphCode would you mind trying to reproduce this with current main to see what happens now?

**Comment by raphCode** [comment]
Confirmed crash on 5cb3fb4a.

**Comment by ondrej-ivanko** [comment]
I had the same error as @raphCode. It started happening since 2 days ago. 

It happens on first zellij startup and then never again. I suspect it might related to  when my laptop ungracefully shutsdown and maybe the `status-bar-tips.cache` is not correctly cleaned up. Also the location in panic msg points to `/tmp/status-bar-tips.cache`, while in fact the cache appears to be in `/tmp/zellij-1000/` dir on my filesystem.

[zellij.log](https://github.com/zellij-org/zellij/files/11284126/zellij.log)

![status-bar-tips.cache](https://user-images.githubusercontent.com/44980578/233309566-95e4a333-7335-42d5-b3b6-0b993dd7ad9c.png)
